src/mongo/dbtests/clienttests.cpp

```diff
@@ -51,7 +53,11 @@ class Base {
         const ServiceContext::UniqueOperationContext opCtxPtr = cc().makeOperationContext();
         OperationContext& opCtx = *opCtxPtr;
         DBDirectClient db(&opCtx);
-
+        auto storagePtr = std::make_unique<repl::StorageInterfaceMock>();
```

Comment:
@benety please review this. As you mentioned NPE should not occur at all and test has to provide instance of ```DropPendingCollectionReaper``` by itself

Reply:
@apocarteres what's the lifetime of "storagePtr"? It looks like its scope is limited to this setup function. You might need to set the singleton for repl::StorageInterface. Look for usages of repl::StorageInterface::set().

Reply:
sure. agree i'm going to update code asap